docs/installing.rst

```diff
@@ -21,7 +19,11 @@ Installing with get-pip.py
 --------------------------
 
 To install pip, securely download `get-pip.py
-<https://bootstrap.pypa.io/get-pip.py>`_. [2]_
+<https://bootstrap.pypa.io/get-pip.py>`_. [2]_ i.e. 
+
+::
+
+ curl https://bootstrap.pypa.io/get-pip.py -o - | python
```

Comment:
I would `curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py`, look at the content of `get-pip.py` with your favorite text editor and make sure it does not do anything harmful, then  execute it with `python get-pip.py`. Arguably it has a binary blob in it, so it's hard to be sure.